nltk/probability.py

```diff
@@ -2143,6 +2124,13 @@ def __gt__(self, other):
             raise_unorderable_types(">", self, other)
         return other < self
 
+    def deepcopy(self):
+        from copy import deepcopy
+
+        return deepcopy(self)
+
+    copy = deepcopy
+
     def __repr__(self):
         """
         Return a string representation of this ``ConditionalFreqDist``.
```

Comment:
Improve efficiency of mathematical operators, see #2938, using (deep)copy and mathematical operators of FreqDist.

Reply:
I think that ought to do it, thanks!